patternfly/patternfly-elements
# Fix issue 1175 cta ie

---
name: Component updates
about: Added css for a tag link color in slotted content, in just IE11.
labels: 
---

## Component updates

Updates relating to a specific component(s).
- pfe-cta


### Related issues

- (#1175) In IE11, the CTAs with the variant primary, possibly when assigned specifically to a slot, do not have a readable type color.


### Preview

Link(s) to demo page(s) where this element can be viewed:
![Screen Shot 2020-11-25 at 12 06 28 PM](https://user-images.githubusercontent.com/3428964/100261103-43507900-2f18-11eb-99bd-f5d4e54c9882.png)

![Screen Shot 2020-11-25 at 12 04 42 PM](https://user-images.githubusercontent.com/3428964/100261114-48152d00-2f18-11eb-9bbe-e17f2a8bcc3d.png)



### What has changed and why

Summarize files edited as part of this MR along with a brief description of what was changed/why.

- updated the a tags within the slotted content to utilize pfe-local(Color).  This provides the correct cascade change the color values based on priority settings.


### Testing instructions

Be sure to include detailed instructions on how your update can be tested by another developer.

1. In an IE11 browser, navigate to https://deploy-preview-1194--happy-galileo-ea79c4.netlify.app/elements/pfe-card/demo/
2. Verify that the CTA button text is visible in all variants of the card.
3. Verify that the link text is still visible on hover as well.


#### Browser requirements

Your component should work in all of the following environments:



### Ready-for-merge Checklist

Check off items as they are completed.  Feel free to delete items if they are not applicable.



### Merging

Please **squash** when merging and ensure your commit message uses [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/#summary) formatting.

**Be sure to share your updates with the [patternfly-elements-contribute@redhat.com](mailto:patternfly-elements-contribute@redhat.com) mailing list!**
